### PR TITLE
Contributor docs

### PR DESCRIPTION
**Pull request by weseldridge**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Code of Conduct
+
+MultiAgentLearning is dedicated to providing a harassment-free experience for everyone, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, age, race, or religion. We do not tolerate harassment of participants in any form.
+
+This code of conduct applies to all MultiAgentLearning spaces (including Gist comments) both online and off. Anyone who violates this code of conduct may be sanctioned or expelled from these spaces at the discretion of the OpenAI team.
```
Comment (cinjon) [inline CODE_OF_CONDUCT.md]: OpenAI --> Pommerman

File: .gitignore [hunk 2]
```diff
@@ -170,6 +170,7 @@ concierge.xcodeproj
 tmpimg/
 local
 .eslintrc
+src/
```
Comment (cinjon) [inline .gitignore]: Reason?

Comment (weseldridge) [reply .gitignore]: running pip install outside of a docker container, it saves gym as a local dep.

Comment (cinjon) [reply .gitignore]: hmm? not quite sure i understand that connection.

Comment (weseldridge) [reply .gitignore]: I guess the way I think about it is... If there is a potential for it to be committed through normal interaction, and we do not want it to be part of the repo it should be ignored. I am good either way. It was just something I noticed when working on the repo. Thoughts?

I can also give more detail also.

Comment (cinjon) [reply .gitignore]: I'm still confused. Where is the "src" directory that this is going to ignore?

Comment (weseldridge) [reply .gitignore]: In the root of the project. `./src/`. A sibling to notebooks and games.

Comment (weseldridge) [reply .gitignore]: I am going to push a change that removes the src from the git ignore. We can continue this outside the scope of this PR. I would rather get the docs into master.

**Comment by cinjon** [comment]
lgtm.

**Pull request merged**